tests/BrowscapTest/Coverage/ProcessorTest.php

```diff
@@ -45,7 +28,7 @@ public function setUp() : void
     /**
      * Data provider for the testJsonStructure test
      *
-     * @return array
+     * @return array[]
```

Comment:
array of arrays? just `array` is probably fine here unless we can be more definitive?

Reply:
changed